.vscode/settings.json

```diff
@@ -6,4 +6,13 @@
 
     // Use 'prettier-eslint' instead of 'prettier'. Other settings will only be fallbacks in case they could not be inferred from eslint rules.
     "prettier.eslintIntegration": true,
+    "search.exclude": {
+        "**/bundle/android": true,
```

Comment:
👋 Enej! I usually exclude the App.js and App.js.map files only, not the whole folder, since those files are big/noisy and usually just are duplicate results found elsewhere. What do you think about going that route? Any particular reason to exclude the whole folder?

Reply:
Nope, let me update it to that we exclude the whole folder.

Reply:
Sorry I miss understood, I updated the code so that we only exclude the App.js and the App.js.map files.